src/test/java/org/broadinstitute/hellbender/utils/HistogramUnitTest.java

```diff
@@ -51,6 +51,15 @@ public void testMedianOfEvens() throws Exception {
         Assert.assertEquals(bimodalHist.median(), 15.0, EPSILON, "");
     }
 
+    @Test
+    public void testMedianEnsureNotHashcodeDependent() throws Exception {
+        Histogram bimodalHist = new Histogram(1d);
+        bimodalHist.add(1.0,2);
+        bimodalHist.add(2.0,2);
+        bimodalHist.add(16.0,2);
+        Assert.assertEquals(bimodalHist.median(), 2.0, EPSILON, "");
+    }
+
     @Test
     public void testMedianOfOdds() throws Exception {
```

Comment:
Also patch `testMedianOfOdds()` so that it would have failed with the previous implementation

Reply:
done